packages/pgsolver/pgsolver.4.4/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis:
+  "A collection of tools for generating, manipulating and - most of all - solving parity games"
+description:
+  "A collection of tools for generating, manipulating and - most of all - solving parity games."
+maintainer: ["Oliver Friedmann" "Martin Lange"]
+authors: ["Oliver Friedmann" "Martin Lange"]
+license: "BSD-3-clause"
+homepage: "https://github.com/tcsprojects/pgsolver"
+bug-reports: "https://github.com/tcsprojects/pgsolver/issues"
+depends: [
+  "ocaml" {>= "4.8"}
+  "dune" {>= "3.10"}
+  "num"
+  "tcs-lib"
+  "extlib"
+  "threads"
```

Comment:
```suggestion
```

There is no such package on the opam-repository.

The OCaml compiler supports threads using a directive (a flag), but there's no opam package needing installation to do so.

Reply:
The error message is:
```
pgsolver.4.4 is not installed. Install it? [Y/n] y
  * Missing dependency:
    - pgsolver >= 4.4 -> threads
    unknown package
```